lib/urlwatch/html2txt.py

```diff
@@ -60,6 +60,13 @@ def html2text(data, method='lynx'):
         d = pyhtml2text.handle(data)
         return d
 
+    if method == 'pyhtml2text_nolinebreak':
```

Comment:
Please add this also to the docstring above, otherwise this feature is not discoverable.